scripts/twister/alt/zephyr/samples/sensor/accel_polling/sample.yaml

```diff
@@ -18,6 +18,21 @@ tests:
       - SHIELD=pca63566
     platform_allow: nrf54h20dk/nrf54h20/cpuapp
 
+  sample.sensor.accel_polling.nrf54h_cpuppr:
+    filter: not CONFIG_COVERAGE
+    sysbuild: true
+    harness_config:
+      fixture: pca63566
+      type: one_line
+      regex:
+        - "^\\s*[0-9A-Za-z_,+-.]*@[0-9A-Fa-f]* \\[m\/s\\^2\\]:    \
+           \\(\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*,\\s*-?[0-9\\.]*\\)$"
+    extra_args:
+      - SHIELD=pca63566
+      - SB_CONFIG_VPR_LAUNCHER=y
```

Comment:
it is actually not needed as the default in `y` so if sysbuild is used for ppr then it will get vpr_launcher https://github.com/nrfconnect/sdk-zephyr/blob/58284ff0e1b85f24d1be928860e9e6eb0e86b4e5/soc/nordic/common/vpr/Kconfig.sysbuild#L6

Reply:
I removed all four occurrences of this KConfig.